### ISSUE TITLE
UnsafeTemplateRegexTests is showing warnings for properly escaped output

### ISSUE THREAD

**Issue by stripthis** [mozilla/amo-validator#568]
The function UnsafeTemplateRegexTests triggers false positive when dealing with EJS templates.
```
   def tests(self):
        for unsafe, safe in (('<%=', '<%-'),
                             ('{{{', '{{'),
                             ('ng-bind-html-unsafe=', 'ng-bind-html')):
            yield self.get_test(...)
```

According to EJS documentation:
- Escaped output with <%= %> (escape function configurable)
- Unescaped raw output with <%- %>

But UnsafeTemplateRegexTests is promoting the opposite. I'm unsure if they are other templating engine that use the exact oposite format (after a quick search I could not find any).

Ref.
- http://ejs.co/ 
- https://github.com/tj/ejs 
- https://github.com/mde/ejs

**Comment by EnTeQuAk** [comment]
Thanks for the report. This sounds weird indeed.

We are pushing towards a world without legacy add-ons and the amo-validator won't be needed in that world so this is currently a very low priority.

**Comment by stripthis** [comment]
@EnTeQuAk fair enough, however this does trigger a lot of warning during the currently manual reviews ( these red flags means amo-reviewers don't review our plugins fast and therefore prevent timely releases). I guess we will just have to replace the default tags to pass these tests.

**Comment by EnTeQuAk** [comment]
Closing this as part of the deprecation process.
